DigitalLearningSolutions.Web/Views/Shared/_LogoutForm.cshtml

```diff
@@ -1,5 +1,7 @@
-﻿<form asp-controller="Logout" asp-action="Index" method="post">
-  <button type="submit" class="nhsuk-button nhsuk-button--secondary">
+﻿@model string

```

Comment:
I think we should use a new `ButtonType` enumeration class which can map a button type to the css class. That'll mean that code which uses _LogoutForm doesn't have to know about the NHS button css classes, and it'll be more obvious what the significance of the model is..

Reply:
Done